Reply/gradle/libs.versions.toml

```diff
@@ -10,20 +10,25 @@ androidx-appcompat = "1.6.1"
 androidx-benchmark = "1.2.0"
 androidx-benchmark-junit4 = "1.2.1"
 androidx-compose-bom = "2024.05.00"
+androidx-compose-material3-adaptive = "1.0.0-beta01"
```

Comment:
You need to make all these changes in the scripts/libs.versions.toml file or else they will be overriden on the next dependency update

Reply:
All good! When I first looked it didn't look like they matched but maybe I was wrong